modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* number
+
+*Default:* `0.2`
+
+---
+
+=== partition_manager_shutdown_watchdog_timeout
+
+A threshold value to detect partitions which shutdown might have been stuck. After this threshold a watchdog in partition manager will log information about partition shutdown not making progress.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `30s`
+
+---
+
+=== pp_sr_smp_max_non_local_requests
+
+Maximum number of x-core requests pending in Panda Proxy and Schema Registry seastar::smp group.  (for more details look at `seastar::smp_service_group` documentation).
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `null`
+
+---
+
+=== quota_manager_gc_sec
+
+Quota manager GC frequency in milliseconds.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `std::chrono::milliseconds(30000)`
+
+---
+
+=== raft_election_timeout_ms
+
+Election timeout expressed in milliseconds.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `1500ms`
+
+---
+
+=== raft_enable_lw_heartbeat
+
+Enables raft optimization of heartbeats.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== raft_flush_timer_interval_ms
+
+Interval of checking partition against the `raft_replica_max_pending_flush_bytes`.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `100ms`
+
+---
+
+=== raft_heartbeat_disconnect_failures
+
+After how many failed heartbeats to forcibly close an unresponsive TCP connection.  Set to 0 to disable force disconnection.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `3`
+
+---
+
+=== raft_heartbeat_interval_ms
+
+Milliseconds for raft leader heartbeats.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `std::chrono::milliseconds(150)`
+
+---
+
+=== raft_heartbeat_timeout_ms
+
+Raft heartbeat RPC timeout.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `3s`
+
+---
+
+=== raft_io_timeout_ms
+
+Raft I/O timeout.
```

Comment:
```suggestion
Raft I/O (input/output) timeout.
```

Reply:
**ENG:** This property is not mentioned anywhere in the docs (except in this properties reference). When/in what context would a user specify this property? Docs needs to provide contextual guidance beyond this entry.